lib/drivers/bayerContourNext.js

```diff
@@ -0,0 +1,510 @@
+/*
+ * == BSD2 LICENSE ==
+ * Copyright (c) 2014, Tidepool Project
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the associated License, which is identical to the BSD 2-Clause
+ * License as published by the Open Source Initiative at opensource.org.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the License for more details.
+ *
+ * You should have received a copy of the License along with this program; if
+ * not, you can obtain one from Tidepool Project at tidepool.org.
+ * == BSD2 LICENSE ==
+ */
+
+/**
+ * IOET's code start here
+ * packet frame format <STX> FN text <ETX> C1 C2 <CR> <LF>
+ * */
+var _ = require('lodash');
+var async = require('async');
+var sundial = require('sundial');
+var crcCalculator = require('../crc.js');
+var struct = require('../struct.js')();
+var annotate = require('../eventAnnotations');
+
+var isBrowser = typeof window !== 'undefined';
+var debug = isBrowser ? require('../bows')('BCNextDriver') : debug;
+
+
+module.exports = function (config) {
+  var cfg = _.clone(config);
+  var hidDevice = config.deviceComms;
+
+  var STX = 0x02;
```

Comment:
technically, STX, ETB, ETX, and EOT are all ASCII_CONTROL characters as well (http://ascii.cl/control-characters.htm). They should probably also go into the collection below.